frame/executive/src/lib.rs

```diff
@@ -466,11 +458,11 @@ where
 
 			let extrinsics_with_author: Vec<(_,_)> = tx_to_be_executed.into_iter().map(|e|
 					(
-						(e.get_address(), e)
+						// its safe to panic here
+						(e.get_account_id(&Default::default()).unwrap(), e)
```

Comment:
Why does Default::default need to be passed in for lookup?

Reply:
Wouldn't a default "lookup" be empty?
How does this work?

Reply:
thats how substrate figures. There is only single implementation of lookup and its generated by macro (construct_runtime) thats how it seems to work

Reply:
you can find similar approach here

https://github.com/mangata-finance/substrate/blob/mangata-dev/frame/executive/src/lib.rs#L387
https://github.com/mangata-finance/substrate/blob/mangata-dev/primitives/runtime/src/generic/unchecked_extrinsic.rs#L126

Reply:
Hmm I see.